conf/default_config.yaml

```diff
@@ -10,7 +10,6 @@
 service_dir: ${ANCHORE_SERVICE_DIR}
 tmp_dir: /analysis_scratch
 log_level: ${ANCHORE_LOG_LEVEL}
-cleanup_images: true
```

Comment:
Why is this removed?

Reply:
I couldn't find any usages in the application, unless I'm missing something.

Reply:
lmk if youw ant me to remove this change

Reply:
This is good. Can leave it.